yjit/src/codegen.rs

```diff
@@ -3959,6 +3943,42 @@ fn gen_send_cfunc(
     asm.cmp(CFP, stack_limit);
     asm.jbe(counted_exit!(ocb, side_exit, send_se_cf_overflow).into());
 
+
+    if flags & VM_CALL_ARGS_SPLAT != 0 {
+        if cfunc_argc > 0 {
+            argc = pop_args_splat(argc, ctx, asm, ocb, side_exit, cfunc_argc as u32);
+        } else {
+            // This is a variadic c function and we'd need to pop
+            // each of the elements off the array, but the array may be dynmically sized
+            gen_counter_incr!(asm, send_args_splat_super);
```

Comment:
Should we rename the counter to `send_args_splat_variadic` or something? The name made me think it had something to do with superclasses?

Reply:
Yeah, sloppy copy and paste. Will fix